Makefile

```diff
@@ -52,5 +52,6 @@ clean:
 .PHONY: dev
 dev:
 	go get github.com/pilu/fresh
-	docker-compose up
+	docker-compose start
```

Comment:
The problem I had initially is that the up only works when you have 
already pulled the image, it seems.

Am 24.06.2016 um 13:40 schrieb Aslak Knutsen:
>
> In Makefile 
> <https://github.com/almighty/almighty-core/pull/4#discussion_r68386598>:
>
> > @@ -52,5 +52,6 @@ clean:
> >  .PHONY: dev
> >  dev:
> >  	go get github.com/pilu/fresh
> > -	docker-compose up
> > +	docker-compose start
>
> @tsmaeder <https://github.com/tsmaeder> The issue you had with 
> docker-compose was that it 'hang'. Up hangs by default, start does 
> not. But they are semantically different as well. 'up -d' sets it in 
> daemon mode so it won't hang. --force-recreate just recreates the 
> container even if the configuration hasn't changed so you start from a 
> clean 'build'.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/almighty/almighty-core/pull/4/files/90e1426b9cf7e73b8bbed3b7aa447293d855f203#r68386598>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AMjc-t28rKU8G879QOGJRNXjTmzjpyfYks5qO8IugaJpZM4I8524>.
>